utils/utils.go

```diff
@@ -77,3 +90,13 @@ func IsResponseEmptyOrMutiple(rLen int, resName string) error {
 	}
 	return nil
 }
+
+func CheckThrottling(err error) *resource.RetryError {
+	rand.Seed(time.Now().UnixNano())
+	if strings.Contains(err.Error(), Throttled) {
+		randTime := (rand.Float32()*(randMax-randMin+1) + randMin) * 1000
```

Comment:
Why "+1" ?

You will get [RandMin, RandMax +1)

Reply:
Do not get 2 times the same number